docs/products/kafka/kafka-connect/howto/jdbc-sink.rst

```diff
@@ -0,0 +1,273 @@
+Create an JDBC sink connector
+=============================
+
+The JDBC (Kava Database Connectivity) sink connector enables you to move data from an Aiven for Apache Kafka® cluster to any relational database offering JDBC drivers like PostgreSQL® or MySQL®. The art
```

Comment:
I am not sure if this part is intentional: The art

Reply:
it wasn't thanks!